boards/arm/nucleo_f103rb/support/openocd.cfg

```diff
@@ -10,3 +12,9 @@ $_TARGETNAME configure -event gdb-detach {
         echo "Debugger detaching: resuming execution"
         resume
 }
+
+rename init old_init
+proc init {} {
+        old_init
+       reset halt
```

Comment:
nit: Something funky going on with the indentation here I think, mix of tab and spaces?

Reply:
Thanks, I've updated with spaces to respect the existing usage.